tasks/handle_registries_conf_d.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: Ensure registries.d exists - system
+  file:
+    path: "{{ __podman_registries_conf_system|dirname }}"
+    state: "directory"
+    owner: "{{ podman_run_as_user }}"
+    group: "{{ podman_run_as_user }}"
+    mode: "0755"
+  when: podman_run_as_user == "root"
+
+- name: Ensure registries.d exists - user
+  file:
+    path: "{{ __podman_registries_conf_non_root|dirname }}"
```

Comment:
This should be `__podman_registries_conf_user` - there should be a test for this

Reply:
Right - if there were a test for `podman_run_as_user: some-non-root-user` then the test would have failed with an undefined variable because `__podman_registries_conf_non_root` is not defined